app/controllers/dois_controller.rb

```diff
@@ -180,8 +190,9 @@ def index
               query: params[:query],
               "provider-id" => params[:provider_id],
               "client-id" => params[:client_id],
-              "page[cursor]" => page[:cursor].present? ? Array.wrap(results.to_a.last[:sort]).first : nil,
-              "page[number]" => page[:cursor].blank? && page[:number].present? ? page[:number] + 1 : nil,
+              # The cursor link should be based on the ES sort, but we want to encode it into a single string
+              "page[cursor]" => show_cursor_link == true ? Base64.encode64(Array.wrap(results.to_a.last[:sort]).join(',')) : nil,
```

Comment:
Consider `strict_encode64` to not add linefeeds.

Reply:
Sounds reasonable, wasn't aware of the difference in Ruby. Will add.